service-loadbalancer/Dockerfile

```diff
@@ -16,10 +16,12 @@
 FROM alpine:3.2
 MAINTAINER Prashanth B <beeps@google.com>
 
-RUN apk add -U haproxy bash curl && \
+RUN apk add -U pcre bash curl socat lua5.3 && \
+  wget -O /var/cache/apk/haproxy.apk https://github.com/aledbf/alpine-build-dev-haproxy/releases/download/1.6-r0/haproxy-1.6-r0.apk && \
```

Comment:
> Can you just pull from haproxy.org and build with the right options 

yes but this will add too many things not related to the kubernetes/contrib project (please check all the required files and packages required to compile the apk)

Just in case I'm using that repo for my alpine images, not just kubernetes/contrib